AdobeStockAsset/Model/OAuth/OAuthException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockAsset\Model\OAuth;
+
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Class OAuthException
+ */
+class OAuthException extends LocalizedException
```

Comment:
Please avoid the introduction of new exception types and pick the most appropriate one from `Magento\Framework\Exception\` namespace